src/transformState.js

```diff
@@ -62,7 +62,27 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  transforms.forEach(move => {
+    switch (move.operation) {
+      case 'addProperties':
+        Object.assign(state, move.properties);
```

Comment:
Do you really need forEach() here? I think that 'for' loop is easier, and you will need less code, without something like 'move.properties);'